### ISSUE TITLE
Wrong example of sliced computation in doc page Numerical Accuracy

### ISSUE THREAD

**Issue by sliorde** [pytorch/pytorch#80940]
### 📚 The doc issue

This issue is related to the page [Numerical Accuracy](https://pytorch.org/docs/stable/notes/numerical_accuracy.html#batched-computations-or-slice-computations)
(Accessed July 6 2022)
(The source of the page in the repository is [here](https://github.com/pytorch/pytorch/blob/master/docs/source/notes/numerical_accuracy.rst#batched-computations-or-slice-computations). But I'm not sure which commit).

Quote from the page:

> let `A` be a 2-dimentional tensor. `A.sum(-1)[0]` is not guaranteed to be bitwise equal to `A[:,0].sum()`.

Viewing `A` as a matrix, notice that `A.sum(-1)[0]` is the sum of the first row, whereas `A[:,0].sum()` is the sum of the first column. These are trivially different, and this difference is not related to numerical accuracy.

See for example:
```python
import torch
A = torch.ones(4,5)
x = A.sum(-1)[0] # equals 5
y = A[:,0].sum() # equals 4
```

I suspect that the intention in this example was either to use `A.sum(0)[0]` for the first calculation (x)or `A[0].sum()` for the second.

### Suggest a potential alternative/fix

I believe that this sentence: 
> `A.sum(-1)[0]` is not guaranteed to be bitwise equal to `A[:,0].sum()`.

should be replaced with one of the following:
> `A.sum(0)[0]` is not guaranteed to be bitwise equal to `A[:,0].sum()`.

> `A.sum(-1)[0]` is not guaranteed to be bitwise equal to `A[0].sum()`.

(which one? depends on the numerical accuracy issue here...)

**Comment by sliorde** [comment]
After testing a bit myself, I think that the documentation can be changed to 

> `A.sum(0)[0]` is not guaranteed to be bitwise equal to `A[:,0].sum()`

Here is an example demonstrating this:

```python
import torch
	
A = torch.zeros(5,4)
A[:,0] = torch.tensor(
	[+0.5223668813705444, 
	 -0.5827842354774475, 
	 +0.24080391228199005, 
	 -0.5065612196922302, 
	 +1.4781107902526855]
)

# these two are not bitwise equal!
x = A.sum(0)[0]
y = A[:,0].sum()

print(x.item()) # 1.1519360542297363
print(y.item()) # 1.1519362926483154
```

(running with the following version
PyTorch version: 1.11.0+cu113
Is debug build: False
CUDA used to build PyTorch: 11.3
ROCM used to build PyTorch: N/A

OS: Ubuntu 18.04.5 LTS (x86_64)
GCC version: (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
Clang version: 6.0.0-1ubuntu2 (tags/RELEASE_600/final)
CMake version: version 3.22.5
Libc version: glibc-2.26

Python version: 3.7.13 (default, Apr 24 2022, 01:04:09) [GCC 7.5.0] (64-bit runtime)
Python platform: Linux-5.4.188+-x86_64-with-Ubuntu-18.04-bionic
Is CUDA available: True
CUDA runtime version: 11.1.105
GPU models and configuration: GPU 0: Tesla T4
Nvidia driver version: 460.32.03
cuDNN version: Probably one of the following:
/usr/lib/x86_64-linux-gnu/libcudnn.so.7.6.5
/usr/lib/x86_64-linux-gnu/libcudnn.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_adv_infer.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_adv_train.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_cnn_infer.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_cnn_train.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_ops_infer.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_ops_train.so.8.0.5
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] numpy==1.21.6
[pip3] torch==1.11.0+cu113
[pip3] torchaudio==0.11.0+cu113
[pip3] torchsummary==1.5.1
[pip3] torchtext==0.12.0
[pip3] torchvision==0.12.0+cu113
[conda] Could not collect
)

**Comment by albanD** [comment]
cc @ngimel who wrote this doc IIRC
